layouts/partials/toc.html

```diff
@@ -5,4 +5,5 @@
 {{ . }}
 {{ end }}
 {{ end }}
-{{ end }}
\ No newline at end of file
+<!-- {{ partial "tags.html" . }} -->
```

Comment:
presumably this could use `partialCached` as it only needs to be constructed once for the whole site